cpu/cpuhotplug.py

```diff
@@ -39,7 +39,7 @@ def collect_dmesg(object):
     object.whiteboard = process.system_output("dmesg").decode('utf-8')
 
 
-class cpustresstest(Test):
+class cpuhotplug(Test):
```

Comment:
@abdhaleegit one request as we are renaming the class name. can we use this opportunity to have the class name as camelCase

Reply:
done